### ISSUE TITLE
Cabal fails after install -- timer_create: Invalid argument

### ISSUE THREAD

**Issue by polygonhell** [Microsoft/BashOnWindows#113]
```
apt-add-repository ppa:hvr/ghc
apt-get update
apt-get install cabal-install-1.24
/opt/cabal/1.24/bin/cabal
```
This results in 
`cabal: timer_create: Invalid argument`

This works on my ubuntu-14 box
ghc appears to actually run, but it's painfully slow to start.

**Comment by voltagex** [comment]
timer_create isn't implemented, see #112

**Comment by parsiya** [comment]
You are correct @voltagex, same thing happens with `pandoc`. Why am I trying to run pandoc in bash on Windows when there is already a Windows package? I have no answer :).
